procedures/katello/branch.md.erb

```diff
@@ -90,6 +90,12 @@
       cp -rf public/apipie-cache/* $GITDIR/theforeman.org/plugins/katello/$VERSION/api
       find $GITDIR/theforeman.org/plugins/katello/$VERSION/api -name "*.json" -type f -delete
       sed -i "/layout: /d" $GITDIR/theforeman.org/plugins/katello/$VERSION/api/index.md
+- [ ] Prepare "Katello Next" and future redmine versions
+  - [ ] Rename the "Katello Next" release to be the newest Katello x.y.0 release (e.g. Katello 4.5.0)
+  - [ ] Recreate the "Katello Next" release and indicate that it is a placeholder for issues belonging to the next version of Katello
+  - [ ] Create the next 'z' release of the latest branched Katello version (e.g. Katello 4.4.1)
+  - [ ] Create the next "Katello x.y.z TODO" custom query using the last one as a template
```

Comment:
is there an example of this TODO query?   i'm not familiar with it, so right off i wouldn't know what to do here.

Reply:
I have the first template one created: https://projects.theforeman.org/projects/katello/issues?query_id=170

Reply:
Do you think it's enough to always have one available for reference under the custom queries?

Reply:
ahh, i think thats fine!

Reply:
What we typically do in Foreman is to have 3 queries. Right now we have 3.0.2, 3.1.2 and 3.2.0. Those are the possible next releases. Once we release 3.1.2, we modify the query to 3.1.3. Once 3.2.0 is released, 3.0.2 is modified to 3.3.0 (since 3.0 is then end of life) and 3.2.0 to 3.2.1.

I know we haven't written this down, but we should. Would this be a good thing to also have for Katello?